node/mdSchema.ts

```diff
@@ -164,4 +172,26 @@ export const schemas = [
       'v-cache': false,
     },
   },
+  {
+    name: B2B_SETTINGS_DATA_ENTITY,
+    version: B2B_SETTINGS_SCHEMA_VERSION,
+    body: {
+      properties: {
+        autoApprove: {
+          type: 'boolean',
+          title: 'Auto Approve',
+        },
+        defaultPaymentTerms: {
+          type: 'array',
+          title: 'Default Payment Terms',
+        },
+        defaultPriceTables: {
+          type: 'array',
+          title: 'Default Price Tables',
+        },
+      },
+      'v-immediate-indexing': true,
+      'v-cache': false,
+    },
+  },
```

Comment:
Given that settings will be stored in vbase rather than MD, this schema is no longer necessary. The constants used by other parts of the code should be moved to `node/constants.md`.

Also, @aurorahey is currently working on a change to the existing `getAppSettings` and `saveAppSettings` GraphQL methods, which will update them to use vbase. I suggest waiting for that release and then adding your new settings to those methods, rather than creating new `getB2BSettings` and `saveB2BSettings` methods.

Reply:
@polishq any ideas on when the release will happen?